credsweeper/credentials/line_data.py

```diff
@@ -191,9 +191,17 @@ def clean_bash_parameters(self) -> None:
                 self.value = value_whsp[0]
 
     def clean_toml_parameters(self) -> None:
-        """Curly brackets may be caught in TOML format"""
-        while self.value.endswith('}') and '{' in self.line[:self.value_start]:
-            self.value = self.value[:-1]
+        """Parenthesis, curly and squared brackets may be caught in TOML format and bash. Simple clearing"""
+        dirty = self.value and self.value[-1] in ['}', ']', ')']
```

Comment:
`dirty` is also good name but IMO it would be good to re-name with like as `is_OOO`.
How about `is_cleared` or `is_clear`?